core/src/main/scala/cats/syntax/foldable.scala

```diff
@@ -158,4 +158,37 @@ final class FoldableOps[F[_], A](val fa: F[A]) extends AnyVal {
       case Some((a, src)) => G.map(p(a))(if (_) Right(Some(a)) else Left(src.value))
       case None => G.pure(Right(None))
     })
+
+  /**
+    * Tear down a subset of this structure using a `PartialFunction`.
+    *{{{
+    * scala> import cats.implicits._
+    * scala> val xs = List(1, 2, 3, 4)
+    * scala> xs.collectFold { case n if n % 2 == 0 => n }
+    * res0: Int = 6
+    *}}}
+    */
+  def collectFold[M: Monoid](f: PartialFunction[A, M])(implicit F: Foldable[F]): M = {
+    val m = Monoid[M]
```

Comment:
totally nitpick - there are two styles of implicit type class instance summoning in this one method (and the `filterFold` method below). I think it would help readability a little bit if we just use `(implicit F: Foldable[F], M: Monoid[M])`

Reply:
This is a nice advice, it will be more readable, and I'll fix it :)

Thanks!